meinberlin/apps/budgeting/assets/ListItemStats.jsx

```diff
@@ -47,8 +53,10 @@ export const ListItemStats = (props) => {
               className="far fa-thumbs-up"
               aria-hidden="true"
             />
-            {props.positiveCount}
-            <span className="visually-hidden">{supportStr}</span>
+            <span aria-hidden="true">{props.positiveCount}</span>
+            <span className="visually-hidden">
+              {props.positiveCount + ' ' + supportText}
```

Comment:
this way it reads the number and stops. then you have to navigate to the next item then it will read out the sentence (without the number). At least in Voiceover.

Reply:
sorry there should have been no span around the positiveCount